src/components/radio-button/radio-button-test.stories.mdx

```diff
@@ -0,0 +1,47 @@
+import { Meta, Story, Preview } from "@storybook/addon-docs";
+import { RadioButton, RadioButtonGroup } from "../radio-button";
+
+<Meta
+  title="RadioButton/Test"
+  parameters={{
+    info: { disable: true },
+    chromatic: {
+      disable: true,
+    },
+  }}
+/>
+
+<Preview>
+  <Story name="with labelHelp and validation props">
+    <RadioButtonGroup
+      name="legend-and-labels-group"
+      onChange={() => console.log("change")}
```

Comment:
Right idea but what we do is use the storybook `action` add-on to output the onChange event to `Actions` tab in Storybook. Take a look at Decimal or Search... In fact, loads of our stories use it, they are just two off the top of my head.